tests/python/unittest/test_tir_schedule_cache_index.py

```diff
@@ -66,13 +65,408 @@ def resize_cache_index(
             B[n, c, vi, vj] = A[n, c, index_var_0[vi, vj], index_var_1[vj]]
 
 
+@T.prim_func
+def bilinear_resize(
+    x: T.Buffer[(1, 3, 40, 40), "float16"], resize: T.Buffer[(1, 3, 80, 80), "float16"]
+):
+    for i0, i1, i2, i3 in T.grid(1, 3, 80, 80):
+        with T.block("resize"):
+            i0_1, i1_1, i2_1, i3_1 = T.axis.remap("SSSS", [i0, i1, i2, i3])
+            T.reads(x[i0_1, i1_1, 0:40, 0:40])
+            T.writes(resize[i0_1, i1_1, i2_1, i3_1])
+            resize[i0_1, i1_1, i2_1, i3_1] = T.Cast(
+                "float16",
+                (
+                    T.Cast(
```

Comment:
Can we have a much simpler case for better readability? Also, please use `#fmt: off` to turn off formatter for complex TVMScript

Reply:
> Can we have a much simpler case for better readability? Also, please use `#fmt: off` to turn off formatter for complex TVMScript

The other test case in the same file is a simpler case, is that too simple? 
What formatter suggest here takes too many lines really, but actually I think it's easier to read in this case.

Reply:
That's fine